### PR TITLE
dlp: add a utility conversion function

### LINKED ISSUE #2321: DLP should use default Java `java.awt.Color` class and not redefine a `Color` class with limited functionalities
```
ImageRedactionConfig imageRedactionConfig = ImageRedactionConfig.newBuilder()
          .setInfoType(infoType)
          .setRedactionColor(Color.newBuilder().setBlue())
          .build();
```

`setRedactionColor` takes in a custom `com.google.privacy.dlp.v2beta1.Color.class` : requiring the use of builders to set RGB values : 
This is a painful user experience.

### PR DESCRIPTION
**Pull request by jeanbza**
There's a lot about this I'm unsure about, since this is my first commit. So, sorry if a lot is wrong here.

Fixes #2321

### CODE REVIEW

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dlp.v2beta1;
+
+import com.google.privacy.dlp.v2beta1.Color;
+
+public class ColorUtil {
```
[the remark this answers is not in the document]
Comment (jadekler) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Done

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  private ColorUtil() {}
+
+  public static final Color convert(java.awt.Color c) {
```
Comment (pongad) [inline google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: I think the name is a little too generic. Maybe `toRpcColor`? That way `toAwtColor` can be the reverse of this and we can add more conversions later. @garrettjonesgoogle Do you have an opinion?

Comment (garrettjonesgoogle) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Yeah, maybe `asDlpColor` might be appropriate.

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dlp.v2beta1;
+
+import com.google.privacy.dlp.v2beta1.Color;
+
+public class ColorUtil {
```
Comment (garrettjonesgoogle) [inline google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: I recommend using the idiomatic name `Colors` for this class.

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  private ColorUtil() {}
+
+  public static final Color convert(java.awt.Color c) {
+    return Color.newBuilder()
+        .setBlue(c.getBlue())
+        .setGreen(c.getGreen())
+        .setRed(c.getRed())
+        .build();
+  }
```
Comment (garrettjonesgoogle) [inline google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Add a test too.

Comment (jadekler) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Done

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java (hunk 3 above)
Comment (jadekler) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Done

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java (hunk 2 above)
Comment (jadekler) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java]: Done

**Comment by jadekler** [comment]
@garrettjonesgoogle PTAL and lmk if anything else to fix up!

File: google-cloud-dlp/src/test/java/com/google/cloud/dlp/v2beta1/ColorTest.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.Test;
+
+public class ColorTest {
+
+  @Test
+  public void asDlpColorTest() {
+    Color got = Colors.asDlpColor(java.awt.Color.GREEN);
+    Assert.assertEquals(got.getBlue(), 0, 0);
+    Assert.assertEquals(got.getGreen(), 255, 0);
+    Assert.assertEquals(got.getRed(), 0, 0);
```
Comment (garrettjonesgoogle) [inline google-cloud-dlp/src/test/java/com/google/cloud/dlp/v2beta1/ColorTest.java]: Don't use the same values in more than one field when you're checking plumbing.

File: google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/Colors.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+ */
+package com.google.cloud.dlp.v2beta1;
+
+import com.google.privacy.dlp.v2beta1.Color;
+
+public class Colors {
+
+  private Colors() {}
+
+  public static final Color asDlpColor(java.awt.Color c) {
```
Comment (garrettjonesgoogle) [inline google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/Colors.java]: If we have a conversion one way, I think we should add one the other way too (for people handling service responses)

Comment (jadekler) [reply google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/Colors.java]: Done

File: google-cloud-dlp/src/test/java/com/google/cloud/dlp/v2beta1/ColorTest.java (hunk 5 above)
Comment (jadekler) [reply google-cloud-dlp/src/test/java/com/google/cloud/dlp/v2beta1/ColorTest.java]: Done

**Comment by jadekler** [comment]
@garrettjonesgoogle PTAL

**Pull request merged**